hcl2template/addrs/plugin.go

```diff
@@ -98,67 +110,71 @@ func IsPluginPartNormalized(str string) (bool, error) {
 //
 // The following are valid source string formats:
 //
-//	name
 //	namespace/name
 //	hostname/namespace/name
 func ParsePluginSourceString(str string) (*Plugin, hcl.Diagnostics) {
-	ret := &Plugin{
-		Hostname:  "",
-		Namespace: "",
-	}
 	var diags hcl.Diagnostics
 
-	// split the source string into individual components
-	parts := strings.Split(str, "/")
-	if len(parts) != 3 {
-		diags = diags.Append(&hcl.Diagnostic{
+	if strings.HasPrefix(str, "/") {
+		return nil, diags.Append(&hcl.Diagnostic{
 			Severity: hcl.DiagError,
-			Summary:  "Invalid plugin source string",
-			Detail:   `The "source" attribute must be in the format "hostname/namespace/name"`,
+			Summary:  "Invalid source URL",
+			Detail:   `The source URL must not start with a '/' character. Ex: "github.com/hashicorp/amazon"`,
```

Comment:
To not give the impression that one cloud is favored over the other lets use the generic happycloud
```suggestion
			Detail:   `The source URL must not start with a '/' character. Ex: "github.com/hashicorp/happycloud"`,
```